### PR TITLE
Wiring up CHBP data

### PR DESCRIPTION
**Pull request by danielduhh**
### Description:

This PR contains various ui and data changes that integrates the CHBP data into the application.

I combined the centroids and polygons into one file ->`chbp_data_2019.json` and uploaded it as a Tileset in Mapbox Studio: https://studio.mapbox.com/tilesets/spatialdev.9w9i8weo/.
I also created a new Style on Studio - https://studio.mapbox.com/styles/spatialdev/cjxyccxk90prc1cpyq11zt62g/. The map was designed after the paper map on CHBP's website -> https://www.capitolhillblockparty.com/festival-map.

Also migrated from css to sass

TODO

- [ ] Need a design for displaying all of the stage information.
- [ ] Need to know list of vendors to show on left menu
- [ ] Wire up right panel lists

### UI images:
Screen Shot 2019-07-14 at 9 02 09 PM

Screen Shot 2019-07-14 at 9 04 23 PM

Screen Shot 2019-07-14 at 9 02 14 PM

### Unit Test Approach:

Describe unit tests being added with that PR.

### Test Results:
- [ ] Did you run `npm test` and did the results pass?
- [ ] Did you add unit tests that cover your changes?

Describe other (non-unit) test results here.

### CODE REVIEW

File: src/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+$blue: #183ca0;
+$red: #dd4e36;
+$gold: #898c45;
+$pink: #f1b1c6;
+$teal: #84cbdd;
```
Comment (cbessee) [inline src/_variables.scss:5]: Nice!

File: src/config.js [hunk 2]
```diff
@@ -5,51 +5,55 @@
...
-      [-122.3144109050901, 47.61535256842217]
-    ]
-  }
-}
+	// Google Analytics
+	ga: {
+		// Global GA identifier
+		id: "UA-XXXXXXXX-X",
+		// Event Category constants
+		event: {
```
Comment (cbessee) [inline src/config.js]: Is all the double tabbing here intentional? Not sure if this is how you want json files done but figured I would call out

**Review by cbessee — CHANGES_REQUESTED** [review]
Looks good! Just a single question about formatting of the GA config. Let me know how you want to handle that and I'll approve!

**Review by stephencerqueira — COMMENTED** [review]
LGTM

File: src/config.js (hunk 2 above)
Comment (danielduhh) [reply src/config.js]: No not at all. Probably something weird with my vs code settings - let me change it back

**Review by cbessee — APPROVED** [review]
Sweet! Thanks!

**Comment by danielduhh** [comment]
Merging this and will work on another PR that addresses the TODO's

**Pull request merged**